openedx/core/djangoapps/user_api/views.py

```diff
@@ -714,27 +714,15 @@ def _add_terms_of_service_field(self, form_desc, required=True):
 
         # Translators: "Terms of service" is a legal document users must agree to
         # in order to register a new account.
-<<<<<<< HEAD
-        label = _(
-            u"I agree to the {platform_name} {terms_of_service} and {privacy_policy}."
-        ).format(
-=======
-        label = _(u"I agree to the {platform_name} {terms_of_service}.").format(
->>>>>>> release-2016-02-09
+        label = _(u"I agree to the {platform_name} {terms_of_service} and {privacy_policy}.").format(
             platform_name=settings.PLATFORM_NAME,
             privacy_policy=privacy_link,
             terms_of_service=terms_link
         )
 
         # Translators: "Terms of service" is a legal document users must agree to
         # in order to register a new account.
-<<<<<<< HEAD
-        error_msg = _(
-            u"You must agree to the {platform_name} {terms_of_service} and {privacy_policy}."
-        ).format(
-=======
-        error_msg = _(u"You must agree to the {platform_name} {terms_of_service}.").format(
->>>>>>> release-2016-02-09
+        error_msg = _(u"You must agree to the {platform_name} {terms_of_service} and {privacy_policy}.").format(
```

Comment:
This is just a formatting issue. For some reason, `gettext` needs the string on the same line as the call to `_()`.